README.md

```diff
@@ -19,44 +19,45 @@ As of 06/11/2017, this project is in the "working prototype" phase of developmen
 
 1. [Raspberry Pi](https://www.raspberrypi.org/products/)
 1. Raspberry Pi [camera module](https://www.raspberrypi.org/products/camera-module-v2/)
-1. SD card with NOOBS (minimum 4GB)
-1. Network access to the Pi (wifi for Pi3, ethernet for Pi2)
+1. SD card with Raspbian GNU/Linux 10 OS (minimum OS and SD Card size 4GB)
+1. Network access to the Pi (WiFi for Pi3, WiFi/ethernet for Pi2)
 1. Suggest wall mountable case for the Pi ([Vilros](http://www.vilros.com/) has a good case with a mount for the camera)
+1. Tactile button and female-to-female jumper lead to get input (By default, missed-moment uses GPIO 26/Pin 37 and the Ground on Pin 39 to attach the button [GPI Reference](https://www.raspberrypi.org/documentation/usage/gpio/)) 
 
 ## Installation
 
-1. Install [NOOBS](https://www.raspberrypi.org/downloads/noobs/) on the Pi
-1. Boot into the Raspberry Pi
-1. Make sure you've enabled the camera module in the settings
-1. NOOBS comes preconfigured with two versions of Python with 2.7 being the default. We use 3.5+, so setup [virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/) in the project directory and switch the python verion
-1. If your Raspberry Pi has 3.4 instead, follow [these instructions](http://wyre-it.co.uk/blog/latestpython/) to install.
-1. open terminal
+1. Download a copy of missed-moment to your local computer
 
     ```
         git clone git@github.com:oudeismetis/missed-moment.git
-        cd missed-moment
-        virtualenv -p /usr/bin/python3.5 venv
     ```
-
-
-1. Create a .env file at the root of the project. (See below for .env contents)
+1. Install the OS on the Pi using method of choice.  e.g. [Raspberry Pi Imager](https://www.raspberrypi.org/documentation/installation/installing-images/) or [NOOBS](https://www.raspberrypi.org/downloads/noobs/) 
+1. Boot into the Raspberry Pi
+1. Make sure you've enabled the camera module in the settings
+1. Missed moment requires Python 3.5+ to interface with the camera.  Raspbian GNU/Linux 10 OS comes with both Python 2.7 and Python 3.7.
```

Comment:
Since Python is a moving target, we should just say that a compatible version of python comes with new installs of Raspberry Pi OS by default.
This project moving forward will not use versions of python that aren't already on the Pi. Been there, done that, it was painful (though necessary back in the 3.4/3.5 days)